pkgs/applications/video/filebot/default.nix

```diff
@@ -10,11 +10,11 @@ let
   };
 in stdenv.mkDerivation (finalAttrs: {
   pname = "filebot";
-  version = "5.1.1";
+  version = "5.1.2";
 
   src = fetchurl {
     url = "https://web.archive.org/web/20230917142929/https://get.filebot.net/filebot/FileBot_${finalAttrs.version}/FileBot_${finalAttrs.version}-portable.tar.xz";
```

Comment:
I guess the URL needs to be updated?

Reply:
Sorry I missed this comment. I'm wondering why this is working. Does web.archive.org redirect to the real location?